src/lib.rs

```diff
@@ -1,3 +1,36 @@
+#![deny(unsafe_code)]
+#![doc(html_logo_url = "https://upload.wikimedia.org/wikipedia/commons/1/12/Bluerobotics-logo.svg")]
+
+/*!
+# Navigator-rs
+
+This crate serves as the entry point for embedding applications using Rust on Blue Robotics's [Navigator](https://bluerobotics.com/store/comm-control-power/control/navigator/).
+
+The Navigator board has the Raspberry Pi HAT form factor, allowing you to attach it to your Raspberry Pi 4 board and unleash the power of your ROV.
```

Comment:
Power of Navigator to develop new solutions or interact with the ROV hardware.

Reply:
Yes

Reply:
did it, please check the new suggested aproach.
@patrickelectric